pyproject.toml

```diff
@@ -14,7 +14,7 @@ classifiers = [
 ]
 requires-python = ">=3.8"
 dependencies = [
-  "opensafely-pipeline @ git+https://github.com/opensafely-core/pipeline@v2024.10.08.124104",
+  "opensafely-pipeline[fastparser] @ git+https://github.com/opensafely-core/pipeline@v2024.10.11.170331",
```

Comment:
Ah sorry, confusingly job-runner _can't_ depend on the `fastparser` extra because it ends up vendored inside opensafely-cli and we can't vendor compiled dependencies. This is part of the Gordian knot we trying to cut by switching to `uv`.

Reply:
Got it, I will switch it back :)

Reply:
We can't hard depend on it for the above reason, but we could add it to the job-runner-dependencies so we get the benefit in job-runner in prod.

Reply:
Ah yeah, I hadn't thought of that.